plugins/com.google.gcp.eclipse.appengine.localserver/gcloud_serverdef.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<tns:ServerRuntime
+   xmlns:tns="http://eclipse.org/jst/server/generic/ServerTypeDefinition"
+   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="http://eclipse.org/jst/server/generic/ServerTypeDefinition ServerTypeDefinitionSchema.xsd"
+   name="GCloud" version="v94.0.0">
+   
+<property id="cloudSdkDirectory"
+  label="Directory where gcloud is installed"
+  type="directory"
+  context="runtime"
+  default="/usr/local/google/home/elharo/google-cloud-sdk" />
+<property id="appEngineSdkDirectory"
+  label="Directory that contains App Engine SDK"
+  type="directory"
+  context="runtime"
+  default="/usr/local/google/home/elharo/appengine-java-sdk" />  
+<property id="serverRootDirectory"
+  label="Runtime Directory"
+  type="directory"
+  context="server"
+  default="/usr/local/google/home/elharo/web" />
+<property id="webappDirectory"
+  label="Web app directory (Directory where appengine-web.xml is found)"
```

Comment:
After testing, the expected value here is the parent of the `WEB-INF` folder which in turn contains the `appengine-web.xml`, the wording here suggest to point to the WEB-INF directory.